src/main/java/random_csapatnev/AmnesiaVirus.java

```diff
@@ -27,11 +27,11 @@ public void Round()
 	 * Felülírja az Agent Effect metódusát, ez a függvény hívódik meg amikor kifejti a hatását karakterre.
 	 */
 	@Override
-	public void Effect(Character c) 
+	public void effect(Character c) 
 	{
 		boolean hasProtection = false;
 		for(Agent a : c.activeAgents) {
-			if(a.name == "AmnesiaVaccine" || a.name == "ProtectiveVaccine") {
+			if("AmnesiaVaccine".equals(a.name) || "ProtectiveVaccine".equals(a.name)) {
```

Comment:
"AmnesiaVaccine" és "ProtectiveVaccine" refaktorálható string literalként szerintem, 
Pl private static final String AMN_VACCINE = ""AmnesiaVaccine"; stb

Reply:
Bennevan több helyen is, akár egy konstansokat tároló külön osztályba is refaktorálható lehet.